image_explorer/image_explorer.py

```diff
@@ -187,6 +193,10 @@ def studio_view(self, context):
         """
         Editing view in Studio
         """
+        xmltree = etree.fromstring(self.data)
+        background = self._get_background(xmltree)
+        if "upload.wikimedia.org/wikipedia" in background['src']:
+            self.hotspot_coordinates_centered = True
```

Comment:
This is a nice idea, but I think we can make it more robust by bumping the [`schema_version` attribute](https://github.com/edx-solutions/xblock-image-explorer/blob/master/image_explorer/image_explorer.py#L41) to `2` and checking whether `schema_version >= 2`.

Also I would prefer to not do this inside `studio_view`.

I suggest doing this:

1. Change the `schema_version` attribute in the default data to `2`.
2. Rename `hotspot_coordinates_centered` xblock field to `_hotspot_coordinates_centered`.
3. Add a new `hotspot_coordinates_centered` python `@property`/method. The method should first check the value of the `_hotspot_coordinates_centered` field, and if `True`, return `True`. Otherwise it should parse the integer value of the `schema_version` attribute from the data field, and return `True` if the value is more than `1`, otherwise return `False`.